src/cromshell/submit/command.py

```diff
@@ -0,0 +1,131 @@
+import click
+import logging
+
+import json
+import requests
+import csv
+import shutil
+import subprocess
+from pathlib import Path
+from pathlib import PurePath
+from datetime import datetime
+
+from cromshell.utilities import cromshellconfig
+from cromshell.utilities import http_utils
+from cromshell.utilities import io_utils
+from cromshell import log
+
+LOGGER = logging.getLogger(__name__)
+
+
+@click.command(name="submit")
+@click.argument("wdl", type=click.Path(exists=True),  required=True)
+@click.argument("wdl_json", type=click.Path(exists=True),  required=True)
+@click.argument("options_json", type=click.Path(exists=True),  required=False)
+@click.argument("dependencies_zip", type=click.Path(exists=True),  required=False)
+@click.pass_obj
+def main(config, wdl, wdl_json, options_json, dependencies_zip):
+    """Submit a workflow and arguments to the Cromwell Server"""
+
+    LOGGER.info("submit")
+
+    io_utils.assert_required_file_is_not_empty(wdl, "WDL")
+    io_utils.assert_required_file_is_not_empty(wdl_json, "Input JSON")
+    io_utils.assert_required_file_is_not_empty(options_json, "Options json")
+    io_utils.assert_required_file_is_not_empty(dependencies_zip, "Dependencies Zip")
+
+    # At this point, we should validate our inputs if we can:
+    womtool_validate_wdl_and_json(wdl, wdl_json)
```

Comment:
How about refactoring these several lines to a function, e.g. something like `validate_input`?

Reply:
done